### DISCUSSION TITLE
Install on Ubuntu 23.10 fails with Run-time dependency python found

### DISCUSSION THREAD

**Discussion by dpickettny** [eoyilmaz/displaycal-py3#311] [Q&A]
Trying to build DisplayCal and everything is good until pip -r install requirements.txt, which produces this error. I went back and installed all the requirements one by one and it fails with trying to install dbus-python; sys_platform=='linux'. From what I can tell, dbus-python is installed. Additionally, the referenced folder in /tmp doesn't exist: 

```
Program dbus-run-session found: YES (/usr/bin/dbus-run-session)
      ../test/meson.build:75: DEPRECATION: Project uses feature that was always broken, and is now deprecated since '1.3.0': str.format: Value other than strings, integers, bools, options, dictionaries and lists thereof..
Run-time dependency python found: NO (tried pkgconfig, pkgconfig and sysconfig)
      
../test/meson.build:98:15: ERROR: Python dependency not found
      
A full log can be found at /tmp/pip-install-xokma425/dbus-python_e268f776efea46dd8cb461383190691a/.mesonpy-npxdxn_d/meson-logs/meson-log.txt
[end of output]
  
  note: This error originates from a subprocess, and is likely not a problem with pip.
error: metadata-generation-failed

× Encountered error while generating package metadata.
╰─> See above for output.

note: This is an issue with the package mentioned above, not pip.
hint: See above for details.
```

**Comment by p5k369** [comment]
Hey @dpickettny,
You are missing the development headers not the python dbus bindings.
Often it is called _dbus-1-dev_ or maybe _dbus-1-devel_ depending on which linux you are on. You probably missing some other dependencies as well, take a look at #264 .

**Reply by dpickettny** [reply]
libX11-devel is installed, as well as the other dependencies listed.

**Reply by p5k369** [reply]
Sorry, I only skimmed over the error message because you previously referred to dbus. You are also missing _python-dev_ or _python3-dev_ (most likely that one for ubuntu) or _python-devel_ or _python3-devel_ depending on which OS you are on.
